app/quest-boost/page.tsx

```diff
@@ -22,7 +22,7 @@ export default function Page() {
   const fetchBoosts = async () => {
     try {
       const res = await getBoosts();
-      setBoosts(res);
+      if (res) setBoosts(res);
```

Comment:
can set this as 
```
if(!res) {
// show error message on screen saying - No boosts available now
(you can use error message screen component in the codebase to do this)
}
setBoosts(res);
```

Reply:
This is the current error message (No quests are being boosted at the moment)

<img width="1430" alt="image" src="https://github.com/starknet-id/starknet.quest/assets/64588227/d2b85aab-0aed-46a8-9b8f-e61a275173c1">

Should I change it to the one below? But then the ErrorScreen component takes the whole page height and there is content below it...

<img width="1431" alt="image" src="https://github.com/starknet-id/starknet.quest/assets/64588227/8cdf644a-5977-4730-b603-5c9cdc733cb9">

Reply:
Hmm , actually what you can do is go with the first option, but also keep the same space in the section as the cards were taking and place the text in the center. Currently it looks a little stuck together vertically. wdyt?

Reply:
makes sense, I was thinking something of the same sort too